Source/Charts/Renderers/CandleStickChartRenderer.swift

```diff
@@ -38,7 +38,7 @@ open class CandleStickChartRenderer: LineScatterCandleRadarRenderer
             accessibleChartElements.append(element)
         }
 
-        for set in candleData.dataSets as! [CandleChartDataSetProtocol] where set.isVisible
+        for case let set as CandleChartDataSetProtocol in candleData where set.isVisible
```

Comment:
do you remember when `xxx case let` is introduced? not sure if people would complain it won't build on older swift

Reply:
Pattern matching has been around since at least Swift 2. I believe It was actually in the first public release.